bodhi/server/models/models.py

```diff
@@ -1649,11 +1649,19 @@ def check_requirements(self, session, settings):
 
         return True, "All checks pass."
 
+    def disable_autopush_for_critical_update(self):
+        """Disable Autopush when critical update receives any negative karma"""
+        log.debug("Disabling Auto Push since the critical update has negative karma")
+        self.autokarma = False
```

Comment:
It looks like this function is only used in one place, which means that we've changed what could have been two lines of code to 1 line of code plus a 4 line function for a total of 5 lines of code. I think this is a case where it makes sense to just put these two lines under the if statement below.